.github/workflows/test_rust.yml

```diff
@@ -163,6 +164,37 @@ jobs:
     runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v5
+        with:
+          # We need history for delta coverage, but we also don't want to fetch
+          # the whole repo. 42 commits seem reasonable
+          fetch-depth: 42
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
```

Comment:
nit: Yet another version to bump manually once in a while (similar to cargo-edit elsewhere)... :face_with_spiral_eyes: (I don't have a better idea though)

Reply:
I'd be against updating this version manually, it's important that diff-cover _works_ and not that it's up to date. If it breaks, it's possible we won't notice that.

I should probably fix diff-cover's version too...